### ISSUE TITLE
TabError: inconsistent use of tabs and spaces in indentation

### ISSUE THREAD

**Issue by cclauss** [IBM/gan-toolkit#1]
Python 3 does not allow mixing of tabs and spaces, spaces are the preferred indentation method.

[flake8](http://flake8.pycqa.org) testing of https://github.com/IBM/gan-toolkit on Python 3.7.0

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./agant/models/generator/cycle_gan.py:30:26: E999 TabError: inconsistent use of tabs and spaces in indentation
    	out_channels = arr[1]
                         ^
1     E999 TabError: inconsistent use of tabs and spaces in indentation
1
```

**Comment by goodboyanush** [comment]
Fixed in this b1c08e2 

@cclauss  Please test and close the issue

**Comment by cclauss** [comment]
Well done.
